docs/content/getting-started.md

```diff
@@ -81,7 +81,7 @@ This has created an empty skeleton with all files you need:
 The generated model for Todo isn't right, it has a user embeded in it but we only want to fetch it if the user actually requested it. So instead lets make a new model in `todo.go`:
 
 ```go
-package gqlgen_todos
+package gqlgen-todos
```

Comment:
package names in go cant have -, they must be valid identifiers. 

import paths can, but the package name cant. Maybe this is a little confusing and we should avoid it all together in the getting started docs.